packages/neos-ui/src/Containers/ContentCanvas/InlineUI/index.js

```diff
@@ -12,26 +12,29 @@ import style from './style.css';
 
 @connect($transform({
     hoveredNode: selectors.CR.Nodes.hoveredSelector,
-    focusedNode: selectors.CR.Nodes.focusedSelector
+    focused: $get('cr.nodes.focused')
```

Comment:
`focusedSelector` returns a node, `$get('cr.nodes.focused')` returns a contextPath and a fusionPath, which are the two values needed to safely identify a rendered node in the DOM.